### ISSUE TITLE
Backwards Compatibility {compatibility with 1.5.2, 1.4.7 etc.}

### ISSUE THREAD

**Issue by Unloaded123456** [Abstrat-Technologies/rustedanvil#11]
That project is getting progressively closer to end, therefore, everyone will can make mods for 1.6.4 without Legacy Fabric. 

**But**: Someone may want even older Forge versions, like Beta 1.7.3 or 1.2.5. If that would be supported, this would please retromodding community and communities like GoldenAgeMinecraft.

**Comment by arek-kwapis** [comment]
Yes, I did think about that. On the master branch, I added a section to the readme that covers exactly this under "Future Plans". It specifically says:

`The end goal of this project is to successfully revive all broken legacy Forge versions, and to potentially future-proof more modern versions (like 1.7.x and 1.8.x)`

I'm also planning on adding channels to the discord so that support can be given for not only RustedAnvil, but also for modding in those legacy versions in general, as the MinecraftForum can sometimes be difficult to follow with everything in one place.

However, on another note, this project's first goal of 1.6.4 modding is **_nowhere near the end_**. There is still a TON of work before it'll be ready, work has just begun really. But at least the important stuff like the github docs are finally done 😁

**Comment by Unloaded123456** [comment]
Luckily, Legacy Forge 1.7.x and 1.8.x are still alive {1.7.10 #1614 requires slight modification of build.gradle, both of them require updating Gradle to 4.4.1}.

**Comment by arek-kwapis** [comment]
Yeah, but for how long? If possible in the future, doing them too would mean that they'll forever work, rather than very possibly becoming the next 965. Proactive versus Reactive 😉
